test/k8s/datapath_configuration.go

```diff
@@ -790,17 +790,16 @@ func monitorConnectivityAcrossNodes(kubectl *helpers.Kubectl) (monitorRes *helpe
 	return monitorRes, monitorCancel, targetIP
 }
 
-func checkMonitorOutput(monitorOutput []byte, egressPktCount, ingressPktCount int) bool {
+func checkMonitorOutput(monitorOutput []byte, egressPktCount, ingressPktCount int) error {
 	// Multiple connection attempts may be made, we need to
 	// narrow down to the last connection close, then match
 	// the ephemeral port + flags to ensure that the
 	// notifications match the table above.
 	egressTCPExpr := `TCP.*DstPort=80.*FIN=true`
 	egressTCPRegex := regexp.MustCompile(egressTCPExpr)
```

Comment:
Not related to your changes but usually it's better to do a regex compilation once if possible (coz it's a pretty heavy operation):
```
var (
	egressTCPRegex = regexp.MustCompile(`TCP.*DstPort=80.*FIN=true`)
	portRegex = regexp.MustCompile(`SrcPort=([0-9]*)`)
)
```
I can do this in separate PR if it makes sense.

Reply:
Go ahead.

Reply:
Will do it after your merge.

Reply:
Never wait :rocket: It shouldn't conflict anyway.